### ISSUE TITLE
A quick question ;-)

### ISSUE THREAD

**Issue by gueniewie** [pdf/freeipa-user-mailalternateaddress#4]
Hello again,

I have found now your work from last month. can I make also only a install.sh when freipa-user-mailaternataddress is installed ?

I have now oracle Linux 8.4 with freeipa 4.9.2 I mean before all is working with centos 7.9

My question is, I mean my dovecot have a Problem to find the mailAternateAddress
have you a example for me for the correct configuration
Or is this with with 4.9.2  no longer working?

Thanks for a answer
by

**Comment by pdf** [comment]
> I have found now your work from last month. can I make also only a install.sh when freipa-user-mailaternataddress is installed?

I'm sorry, I don't quite understand your question, could you rephrase?

> I have now oracle Linux 8.4 with freeipa 4.9.2 I mean before all is working with centos 7.9

I'm not sure about Oracle Linux, but I've used the installer on Rocky Linux 8.4 with FreeIPA 4.9.2 without any issues.

**Comment by gueniewie** [comment]
Hello,
I wanted to ask if an older version of the script was already installed, whether it is possible to simply transfer the newer version?
Or could there be problems with that?

HELP: ;-)

I use Dovecot for my emails but unfortunately the old LDAP settings no longer work. I can no longer read the mailAlternatAdrress?

Can you help with the syntax?
Or if you have an example to make it work again

**Comment by pdf** [comment]
> I wanted to ask if an older version of the script was already installed, whether it is possible to simply transfer the newer version?

You may want to remove the old versions of `user_mailalternateaddress.js` and `user_mailalternateaddress.py` if you're upgrading from a very old version of FreeIPA (< v4.6.x), otherwise it should be fine to run the installer again to overwrite the previous versions with the new ones.

> I use Dovecot for my emails but unfortunately the old LDAP settings no longer work. I can no longer read the mailAlternatAdrress?

Your user attributes should not be affected by upgrading FreeIPA, so I don't know why you would be having problems with Dovecot. I would suggest using `ldapsearch` to make sure your directory works properly, and that your filters are right.
